site/src/components/JsonSchemaPreview.tsx

```diff
@@ -128,6 +129,37 @@ function Schema({ schema: baseSchema, defs, parent, foldable }: SchemaProps) {
     );
   }
 
+  // Intersection
+  if (schema.anyOf) {
+    const { anyOf, ...common } = schema;
```

Comment:
`common` doesnt exist on `JSONSchema7`, you'll have to use the same method that the `oneOf` branch uses

Reply:
You're right, i totally misread the destructuring. In any case the two branches should match so you can update `oneOf` to be consistent